docs/UserGuide.md

```diff
@@ -348,6 +349,13 @@ Furthermore, certain edits can cause the MyBookshelf to behave in unexpected way
 1. `add` command is used to record new user's personal information into the Contact List.
 1. `add` and `edit` Command **DOES NOT** support adding/editing merit score or borrowed book.
 1. `add`, `delete`, `edit`, `clear` and `find` commands are for managing users, while `addbook`, `delbook`, `borrow`, `donate` and `return` commands are for managing books.
+1. `borrow` command is cases-sensitive to allow books of similar titles to be differentiated.
+1. `delbook` deletes one book at a time to prevent accidental deletion of  all entries with the same book  title.
+1. Email and phone numbers have not been made unique identifiers as there may be case scenarios where  two users might share the same email or phone number such as small child and a parent.
+1. Book title is used instead of index as there is no urgent need for current functions like `borrow` or `addbook`. Furthermore, there is a plant to implement a findbook function in the future which will require use of book title. If booktitle is the same, include author name or other distinguishing features in brackets to differentiate book under `donate` and `addbook` commands.
+1. `delete` command will delete users along with any books borrowed. This is because community library manager cannot retrieve the books. If books are returned, community library manager can use `addbook` command to include the books back into the list.
+1. There is no constrain for book titles as it can be anything. For example, it could be in different languages like Chinese or Arabic.
```

Comment:
I think "constrain" should be spelt as "constraints"

I think there is a grammar error here for book titles and it.
I think it should be either
There **are** no constraints for book titles as **they** can be anything.
There **is** no constraint for book titles as **it** can be anything.